package/Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 WORKDIR /var/submariner
 
 COPY strongswan*.tar.gz /var/tmp
 
-RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && apt-get -y update && apt-get -y install curl iproute2 iptables libatm1 libgmp10 libmnl0 libnfnetlink0 libssl1.0.0
+ENV DEBIAN_FRONTEND=noninteractive
 
-RUN /bin/bash -c "cd / && cp /var/tmp/strongswan*.tar.gz . && tar -zxvf strongswan*.tar.gz && rm strongswan*.tar.gz"
+RUN apt-get -y update
+RUN apt-get -y install curl iproute2 iptables libatm1 libgmp10
```

Comment:
I’m not sure why `curl` and `libgmp10` are needed, but I left them in to err on the safe side. I’ll revisit this once we switch to StrongSwan packages (since that will take care of those dependencies).

Reply:
Can the 2 RUNs be combined?